src/Mechanical/Rotational/components.jl

```diff
@@ -107,4 +126,38 @@ function IdealGear(;name, ratio, use_support=false)
         0 ~ ratio*flange_a.tau + flange_b.tau
     ]
     extend(ODESystem(eqs, t, sts, [ratio]; name=name), partial_element)
+end
+
+"""
+    RotationalFriction(;name, f=0.001, tau_c=20, w_brk=0.1, tau_brk=25) 
+
+Models rotational friction with Stribeck friction, Coulomb friction and viscous friction between the two flanges.
```

Comment:
```suggestion
Models rotational friction with Stribeck effect, Coulomb friction and viscous friction between the two flanges.
```
just a suggestion to reflect typical usage in literature.